package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "npx lerna bootstrap",
```

Comment:
I think this part isn't necessary though

Reply:
https://github.com/trufflesuite/truffle/pull/2683/checks?check_run_id=347793212#step:5:6 😕